common/src/main/java/com/microsoft/identity/common/internal/fido/FidoChallengeFactory.kt

```diff
@@ -127,5 +129,23 @@ class FidoChallengeFactory {
             }
             return param
         }
+
+        /**
+         * If parameter is empty, replace parameter value with given default value.
+         * @param field passkey protocol field
+         * @param value value for a passkey protocol parameter
+         * @param defaultValue value to be used as default
+         * @return validated parameter value, or default value if initial value is null.
+         * @throws ClientException if the parameter is empty
+         */
+        @Throws(ClientException::class)
+        fun validateParameterOrReturnDefault(field: String, value: String?, defaultValue: String): String {
```

Comment:
private or internal?

Reply:
And other validate*methods

Reply:
Added internal to all validate methods: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2172/commits/ada80ee79a65f8d71d54e39ce234c9ee3f7358b1